login.html

```diff
@@ -184,6 +186,8 @@
     }
   }
   function onError(errorObj) {
+    email.value = "";
```

Comment:
You don't want to do this if, say, the password is too short.

You either need to compare errorObj to UNKNOWN_ERROR, or do this where you call errorFunction(UNKNOWN_ERROR)